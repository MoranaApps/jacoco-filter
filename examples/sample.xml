<?xml version="1.0" encoding="UTF-8"?>
<report name="Test Report">
    <package name="com/example">
        <class name="com/example/MyClass" sourcefilename="MyClass.java">
            <method name="doWork" desc="()V" line="10">
                <counter type="INSTRUCTION" missed="5" covered="10"/>
                <counter type="LINE" missed="1" covered="3"/>
            </method>
            <method name="getData" desc="()Ljava/lang/String;" line="20">
                <counter type="INSTRUCTION" missed="2" covered="8"/>
            </method>
            <counter type="INSTRUCTION" missed="7" covered="18"/>
            <counter type="LINE" missed="1" covered="3"/>
        </class>
        <counter type="INSTRUCTION" missed="7" covered="18"/>
    </package>
</report>
