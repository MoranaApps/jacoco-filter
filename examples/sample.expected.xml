<?xml version='1.0' encoding='UTF-8'?>
<report name="Test Report">
    <counter type="INSTRUCTION" missed="0" covered="0"/>
    <counter type="LINE" missed="0" covered="0"/>
    <counter type="METHOD" missed="0" covered="0"/>
</report>
